src/seedsigner/views/seed_views.py

```diff
@@ -2294,3 +2301,289 @@ def run(self):
 
         # Exiting/Canceling the QR display screen always returns Home
         return Destination(MainMenuView, skip_current_view=True)
+
+
+
+"""****************************************************************************
+    Shamir's Secret Sharing Views
+****************************************************************************"""
+class SeedEntryShamirThresholdView(View):
+    def __init__(self):
+        super().__init__()
+
+
+    def run(self):
+        title = _("SLIP-39 Threshold")
```

Comment:
I think we'll need to standardize on just one way to refer to Shamir / SSS / SLIP-39 in the UX.

My guess is that "Shamir" should be our default term, unless there are specific cases where one of the others is better.